### ISSUE TITLE
Remove empty paths filter throws null error

### ISSUE THREAD

**Issue by wsocha** [thim81/openapi-format#96]
Hi! I tried to use the package to filter my openapi schema but I'm getting `TypeError: Cannot read properties of null (reading 'includes')` from [this line in openapi-format.js](https://github.com/thim81/openapi-format/blob/1355fc1f930cb8eadbe0fbd407bf04a4d8edfa81/openapi-format.js#L492).
```
if (this.parent && this.parent.key === 'paths' && !httpVerbs.some(i => this.keys.includes(i))) {
```
In my api specification I have a request body schema that has `paths` among its properties so when this line of code is hit, `this.key` is "type", `this.parent.key` is "paths" but `this.keys` is null and thats why the null exception. I guess you wanted this validation to be run only when traversing paths=endpoints, not some `paths` property. 
Some more strict check should be made to make sure it's only executed when it's on the "first-level" `paths`, e.g.:
```
this.parent.path.length === 1  && this.parent.path[0] === 'paths' // this.parent.path === ['paths']
```

**Comment by thim81** [comment]
hi @wsocha 

Thanks for reporting this issue.
I have just released [version 1.15.2](https://github.com/thim81/openapi-format/releases/tag/v1.15.2), which contains a fix for this error.

**Comment by wsocha** [comment]
Hi @thim81, thank you very much for the fix and the release.
